src/views/Jobs/Jobs.js

```diff
@@ -79,7 +79,7 @@ function Jobs({
             <Paneset id="jobs-paneset">
               <JobsSearchPane
                 {...sasqParams}
-                defaultWidth="20%"
+                defaultWidth="25%"
```

Comment:
I increased this a bit because the datepicker would not correctly display on the screen when opened.